app/front/scripts/config/config.js

```diff
@@ -21,18 +22,26 @@ angular.module('Application')
     }
   ])
   .run([
-    '$rootScope', '$location',
-    function($rootScope, $location) {
+    '$rootScope', '$location', 'Configuration',
+    function($rootScope, $location, Configuration) {
       $rootScope.isLoading = {
         application: true
       };
       var lang = $location.search().lang;
       $rootScope._t = i18n.init(lang);
+
+      // "Suffix": scale, "Suffix 2": scale2
+      var scale = $rootScope._t('Value Formatting Scale');
+      if (scale == 'Value Formatting Scale') {
+        scale = '"Billions":1000000000,"Millions":1000000,"Thousands":1000';
```

Comment:
Let's move the defaults to the `visualizations.formatValue`, in case there's a parsing problem.